### ISSUE TITLE
Support wildcard imports

### ISSUE THREAD

**Issue by foges** [euberdeveloper/esm-module-alias#6]
Currently you need to specify the alias for each module, which can be quite cumbersome if you have multiple modules. It would be great if you could support wildcard imports. Something as easy as replacing

```
  const isAliasInSpecifier = (path, alias) => {
    return path.indexOf(alias) === 0
      && (path.length === alias.length || path[alias.length] === '/');
  };
```

with this works

```
  const isAliasInSpecifier = (path, alias) => {
    return path.indexOf(alias) === 0
  };
```

**Comment by euberdeveloper** [comment]
Can you give me an example, in the esm code, of wildcard imports that gives an error with the current solution?

**Comment by foges** [comment]
My resolve code looks like this

```
export const resolve = loadAliases({
  "@/": "dist/src/",
})
```

In my esm code I then have

```
import client from "@/client"
```

esm-module-alias then fails to substitute the alias because `path = "@/client"` and `alias = "@/"`. 

I suppose the code works if you have

```
export const resolve = loadAliases({
  "@": "dist/src/",
})
```

But from a semantic point of view I don't think that's particularly clean, since I wouldn't want `@client` to resolve to `dist/src/client`, which is what it reads like. An alternative could be to change it to

```
 const isAliasInSpecifier = (path: string, alias: string) => {
    return (
      path.indexOf(alias) === 0 &&
      (path.length === alias.length || path[alias.length] === "/"  || path[alias.length - 1] === "/")
    )
  }
```

**Comment by euberdeveloper** [comment]
The purpose of this lib was having a way to basically keep the same code of a project using **module-alias** and **commonjs** but passing to **esm**. Since this seems to me to be the behaviour of **module-alias**, I will not change it. 

Maybe later I will think about a perfect behaviour for the path-matching-alias part.

What I will probably do now instead is allowing users to customize the matching behaviour, so that you can change the function as you wish.

I'll write a comment here when the feature is in production.

Thanks for the issue,
Eugenio

**Comment by euberdeveloper** [comment]
Fixed by PR #7
